packages/fds-components/src/components/buttons/ButtonGroup.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import GroupButton from './GroupButton';
+
+/* Allows you to combine functions */
+function combine(...functions) {
+  return (...args) =>
+    functions
+      .filter((func) => typeof func === 'function')
+      .forEach((func) => func(...args));
+}
+
+const ButtonGroup = ({ activeValue, buttons, onChange, className, ...restBtnGroup }) => {
+  const rootClass = cx('btngroup', className);
+  return (
+    <div className={rootClass} {...restBtnGroup}>
+      {buttons.map(({ value, content, icon, onClick, ...restBtn }) => (
+        <GroupButton
+          key={value}
+          isActive={activeValue === value}
+          onClick={combine(() => onChange(value), onClick)}
+          label={content || value}
+          Icon={icon}
+          {...restBtn}
+        />
+      ))}
+    </div>
+  );
+};
+
+ButtonGroup.propTypes = {
+  activeValue: PropTypes.string,
+  /** Each object in array accepts value, content, onClick, and icon, and also will pass through any extra props */
+  buttons: PropTypes.arrayOf(
+    PropTypes.shape({
+      value: PropTypes.string,
```

Comment:
I would add an isRequired here since it's required for both UI and performance (`key` value). That'll also help us understand that it is valued higher than `content`

Reply:
Good call. Thanks!